### ISSUE TITLE
[Settings Editor] Cannot shift focus back to TOC

### ISSUE THREAD

**Issue by 9at8** [microsoft/vscode#101959]
When I'm navigating the settings in the settings editor, there's no way to move focus back to the TOC tree. (Other than shift+tabbing through all the settings)

To repro:

1. Open the settings editor
2. Scroll down for a bit
3. Only way to go to TOC on the left is to shift + tab all the way back to the top

**Comment by alexr00** [comment]
How do I get back to the TOC without shift+tab all the way back up to the top?

**Comment by 9at8** [comment]
@alexr00 if you're using a screen reader, then you can use browse mode to navigate to the previous landmark, and that will focus the TOC. But I have been working on adding a default keybinding to it.

**Comment by 9at8** [comment]
Actually now that I think about it, it's a duplicate of #92935
